modules/administration/pages/maintenance-window.adoc

```diff
@@ -53,8 +53,10 @@ complete procedure, also see above:
 By default, several update channels are configured and enabled for the {susemgr} Server.
 New and updated packages will become available automatically.
 
+ifeval::[{suma-content} == true]
 To keep {susemgr} up to date, either connect it directly to {scc} or use {rmtool} (RMT).
 You can use RMT as a local installation source for disconnected environments.
+endif::[]
 
 You can check that the update channels are available on your system with this command:
 
```

Comment:
Unrelated to this PR?

The content below only applies to SUSE Manager, not Uyuni. As we neither have MUs, I'd say you could move the `endif::[]` so it applies until line 99.

Reply:
ok, but ifdefs must not exceed more than 1 page (~20 lines) ;)  That's my very personal rule. I either will create an include file or create 1 file file for suma and the other one for uyuni (and put the ifdef in the upper-level nav-file).

Reply:
done